docs/how-to/cluster/migrate.md

```diff
@@ -13,34 +15,43 @@ For a brief explanation of how MirrorMaker works, see the [MirrorMaker explanati
 
 To migrate a cluster we need:
 
-- An "old" existing Apache Kafka cluster to migrate from.
-  - The cluster needs to be reachable from/to the new Apache Kafka cluster. 
-- A bootstrapped Juju VM cloud running Charmed Apache Kafka to migrate to. For guidance on how to deploy a new Charmed Apache Kafka, see:
+- An "old" existing Kafka cluster to migrate from.
+  - The cluster needs to be reachable from/to the new Charmed Apache Kafka Connect cluster. 
+- A bootstrapped Juju VM cloud
+- A Charmed Apache Kafka Connect cluster to run the MirrorMaker tasks
+    - <!-- TODO: add Kafka Connect docs link here? -->
```

Comment:
Todo: Right now this comment is an empty sub-element of the list. If you remove the hyphen it will become a dedicated line, which will separate the list into two lists. The best way is to add a comment after the end of a list, not in the middle of it.